dae/dae/autism_gene_profile/db.py

```diff
@@ -80,7 +80,9 @@ def get_agp(self, gene_symbol) -> Optional[AGPStatistic]:
         """
         table = self.agp_table
         query = table.select()
-        query = query.where(table.c.symbol_name == gene_symbol)
+        query = query.where(
+            func.lower(table.c.symbol_name) == func.lower(gene_symbol)
```

Comment:
You could use the native Python `.lower()` for `gene_symbol` since it's a simple `str` instance